Client/src/Components/Authentication/LoginView.tsx

```diff
@@ -125,7 +155,7 @@ export default function LoginView() {
                 </Button>
                 <Grid container>
                   <Grid item xs>
-                    <Link href="#" variant="body2">
+                    <Link onClick={handleOpen} variant="body2">
```

Comment:
shouldnt be a button?

Reply:
![image](https://user-images.githubusercontent.com/6309685/106389756-16ef2800-63b3-11eb-8abd-5772844d98b7.png)
No, link should be fine

Reply:
i see